.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+# Contributing to Mobify Code Style
+
+## Bugs, Feature Requests
+Submit an [issue](https://github.com/mobify/mobify-code-style/issues).
+
+## Pull Requests
+
+1. Make a new branch from `develop`.
```

Comment:
I'm good with keeping it as release deployment and you taking release manager duties. :)